components/treeview/index.css

```diff
@@ -154,3 +157,8 @@ governing permissions and limitations under the License.
 .spectrum-TreeView-item--indent10 {
   padding-inline-start: calc(10 * var(--spectrum-treeview-item-indent));
 }
+
+.spectrum-TreeView-heading {
+  padding: calc(calc(var(--spectrum-treeview-item-margin-y) * 2) + var(--spectrum-treeview-item-padding-y)) 10px;
```

Comment:
maybe refactor this in multiple lines for readability? took me a second to see that the 10px is not part of the calc :).
Is it possible to get the 10px from some DNA var?

Reply:
+1 on finding out why that `10px` is hardcoded.